builtin/providers/datadog/resource_datadog_monitor.go

```diff
@@ -0,0 +1,233 @@
+package datadog
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+const (
+	MONITOR_ENDPOINT = "https://app.datadoghq.com/api/v1/monitor"
```

Comment:
Go style nit - constants should use `UpperCamelCase` if they are exported and `lowerCamelCase` if they are internal to the package